packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -75,7 +76,10 @@ class ClayMultiSelect extends ClayComponent {
 	_defaultAutocompleteItemSelected(event) {
 		const {label, value} = event.data.item;
 
-		const item = this._createItemObject(label, value);
+		const item = this._createItemObject(
+			htmlEscape(label),
+			htmlEscape(value)
```

Comment:
@matuzalemsteles @jbalsas 
I'm not very familiar with Clay, but this doesn't seem like the right fix to me. 

1. You're basically escaping too early. While there might not be any problems right now, this is going to create problems later when `item` gets used slightly differently from how it's used now. I've seen this problem many times now. Instead of escaping here, you should delay the escape until right before the value is outputted to the DOM.

2. You cannot use the same escape method for label and value. From what I can tell, label is used between `<span>` and `</span>` where you need something like `HtmlUtil.escape()`. For value, this is used in a tag's attribute (e.g., `<input value="value">` where you need something like `HtmlUtil.escapeAttribute()`

Reply:
About 

1. I think this is the last place where we can actually scape before this gets passed down to the template engine to be committed to the DOM, so I don't think there's other place we can do it.
2. That's correct, we should have a different scaping for `value` which goes to an attribute.

Reply:
> You cannot use the same escape method for label and value. From what I can tell, label is used between <span> and </span> where you need something like HtmlUtil.escape(). For value, this is used in a tag's attribute (e.g., <input value="value"> where you need something like HtmlUtil.escapeAttribute()

This is a particular case as `label` and `value` here are the same things at the end and we may not even need to add an escape here because these values ​​are from the autocomplete that comes from the backend data.

Reply:
We commit `label` to the DOM inside html and `value` to the DOM inside an HTML attribute and SOY is failing to escape those for us. I'm not sure why that happens, but since we're migrating to the React components, this is meant to be a simple temporary mitigation.

Reply:
Guys, please don't forget this is going to go into 7.2 patch and will stay with us for long, it's not that temporary as it seems. If you aim to make it more stable, please do it now. Thanks! :)

Reply:
We're planning to update our component usage in 7.2, so it is temporary because we're looking to use more robust components there too.

Reply:
I see, nice then. Thanks.